examples/seismic/tti/operators.py

```diff
@@ -368,11 +385,11 @@ def ForwardOperator(model, source, receiver, space_order=4,
        :param: time_order: Time discretization order
        :param: spc_order: Space discretization order
        """
+    if kernel == 'staggered':
```

Comment:
I think the caller should call one or the other

Reply:
no...?

Reply:
Not sure how to do it nicely

Reply:
how do you mean? the caller can use either ``ForwardOperator`` or ``ForwardOperatorStagg``, can't he?